tests/utilities/test_reader.py

```diff
@@ -532,6 +532,14 @@ def test_openfoam_decompose_polyhedra():
     assert reader.decompose_polyhedra is True
 
 
+def test_openfoam_skip_zero_time():
+    reader = get_cavity_reader()
+    reader.skip_zero_time = False
+    assert reader.skip_zero_time is False
+    reader.skip_zero_time = True
```

Comment:
It would be nice to test that `reader.time_values` contains zero when `reader.skip_zero_time is False`, but does not contain zero when `reader.skip_zero_time is True`